daemons/dss-notify-v2/app.py

```diff
@@ -0,0 +1,86 @@
+"""
+See Readme.md in this directory for documentation on the dss-notify-basic daemon.
```

Comment:
`dss-notify-basic` should be changed to `dss-notify-v2` once the `Readme.md` is made.

Reply:
will do